ocaml/xapi/network.ml

```diff
@@ -42,6 +42,9 @@ let transform_networkd_exn pif f =
 	| Not_implemented ->
 		let e = "networkd function not implemented" in
 		reraise Api_errors.pif_configuration_error [Ref.string_of pif; e]
+	| Vlan_in_use params ->
+		let e = String.concat "; " (List.map (fun (k, v) -> k ^ " = " ^ v) params) in
+		reraise Api_errors.vlan_in_use [Ref.string_of pif; e]
```

Comment:
I don't think the PIF ref is appropriate here. The datamodel says that the first param is the device. Also, if you change the arguments of the networkd exception to `(string * int)` (see xapi-project/xcp-idl#122), then you can just do:
```
| Vlan_in_use (device, vlan) ->
    reraise Api_errors.vlan_in_use [device; vlan]
```